### DISCUSSION TITLE
[Feature] Find geo data by address field

### DISCUSSION THREAD

**Discussion by Flyingfufu** [immich-app/immich#7387] [Feature Request]
### The feature

First, very pleased to have found Immich... I love it, so thanks for the hard work !

I went trhough all and didn't find this feature request, hope I didn't overlook it.

When looking to add the geo position to some pictures we currently have to look in the map a place the mark on it.
It would be awesome is an address field would exist in which an exact address, name of a city or POI cold be entered. It would be sometime quicker and definitively more precise...

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
This just got merged last night: #7139

**Reply by Flyingfufu** [reply]
Was not searching the right words, therefore missed it.; sorry for that !

Thanks, looking fwd to it in the next release ! Does it work also when multiple files are selected or only single files are edited ?
